src/qibojit/backends/gpu.py

```diff
@@ -473,7 +473,14 @@ def calculate_expectation_density_matrix(self, matrix, state, normalize):
             ev = ev / norm
         return ev
 
-    def calculate_eigenvalues(self, matrix, k=6):
+    def calculate_eigenvalues(self, matrix, k: int = 6, hermitian: bool = True):
+        if not hermitian:
+            raise_error(
+                NotImplementedError,
+                "GPU backends do not support `np.linalg.eig` "
+                + "or `np.linalg.eigvals` for non-Hermitian matrices.",
+            )
```

Comment:
Should we fall back to CPU in those cases rather than crashing? Maybe something like:
``` python
if not hermitian: 
    super().calculate_eigenvalues(matrix.numpy(), k, hermitian)
```

Reply:
It could be an option. For now, I just moved the error message from `qibo` to here. If it's bet to default to `cpu`, I'd say a warning message is needed.